### PR TITLE
Update formatting in DG

### PR DESCRIPTION
**Pull request by ethantyh692**

### CODE REVIEW

**Review by Chronoxy — APPROVED** [review]
LGTM 👍

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -338,18 +338,18 @@ _{Explain here how the data archiving feature will be implemented}_
...
 **Target user profile**:
 
-* freelance pre-university home tutors
-* has a need to manage a significant number of students, their contacts and relevant information
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Freelance pre-university home tutors
+* Has a need to manage a significant number of students, their contacts and relevant information
```
Comment (Chronoxy) [inline docs/DeveloperGuide.md:342]: Maybe it might be better to be
"Manages a significant number of students"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -338,18 +338,18 @@ _{Explain here how the data archiving feature will be implemented}_
...
+* Prefer desktop apps over other types
```
Comment (Chronoxy) [inline docs/DeveloperGuide.md:343]: What do you think of "Prefers desktop applications" instead?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -338,18 +338,18 @@ _{Explain here how the data archiving feature will be implemented}_
...
+* Can type fast
```
Comment (Chronoxy) [inline docs/DeveloperGuide.md:344]: I think this might not be required, since we mentioned "Prefers typing rather than mouse interactions"

**Review by Chronoxy — COMMENTED** [review]
Some ideas, but overall LGTM 👍

**Review by NorbertLoh — APPROVED** [review]
LGTM!

**Pull request merged**